chsdi/templates/htmlpopup/geschiebemessnetz.mako

```diff
@@ -0,0 +1,164 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+<% c['stable_id'] = True %>
+<%
+    lang = 'fr' if lang in ('fr', 'it') else 'de'
+%>
+<tr><td class="cell-left">${_('ch.bafu.feststoffe-geschiebemessnetz.gsch_n')}</td><td>${c['attributes']['gsch_n']}</td></tr>
+<tr><td class="cell-left">${_('ch.bafu.feststoffe-geschiebemessnetz.geologie')}</td><td>${c['attributes']['geologie']}</td></tr>
+<tr><td class="cell-left">${_('ch.bafu.feststoffe-geschiebemessnetz.fluss')}</td><td>${c['attributes']['fluss']}</td></tr>
+<tr><td class="cell-left">${_('ch.bafu.feststoffe-geschiebemessnetz.station')}</td><td>${c['attributes']['station']}</td></tr>
+<tr><td class="cell-left">${_('ch.bafu.feststoffe-geschiebemessnetz.institut')}</td><td>${c['attributes']['institut']}</td></tr>
+<tr><td class="cell-left">${_('ch.bafu.feststoffe-geschiebemessnetz.amt')}</td><td>${c['attributes']['amt']}</td></tr>
+<tr><td class="cell-left">${_('link')}</td>
+    <td>
+    % if c['attributes']['pdf_file']:
```

Comment:
In other makos, we check for existence. Here we assume that pdf exists if specified in db. Correct?